lib/scoped_search/validators.rb

```diff
@@ -0,0 +1,7 @@
+module ScopedSearch
```

Comment:
It will be used by the consumers when defining the validation for the fields:

``` ruby
scoped_search :on => :owner_id,  :complete_enabled => false, :only_explicit => true, :validator => ScopedSerach::Validators.NUMERIC
```